web-bundle/src/main/java/com/graphhopper/replica/GtfsLinkMapper.java

```diff
@@ -199,7 +199,7 @@ private List<String> getGtfsLinkCsvRowsForFeed(SetMultimap<String, Pair<Stop, St
     private static String getCsvLine(String routeId, String feedId, String stopId, String nextStopId,
                                      double stopLat, double stopLon, double stopLatNext, double stopLonNext,
                                      String stableEdgeIdString, String transitEdgeString) {
-        return String.format("%s,%s,%s,%s,%f,%f,%f,%f,%s,%s", routeId, feedId, stopId, nextStopId,
+        return String.format("\"%s\",\"%s\",\"%s\",\"%s\",%f,%f,%f,%f,\"%s\",\"%s\"", routeId, feedId, stopId, nextStopId,
```

Comment:
worth a code comment about why we do this escaping?

Reply:
Good call, added!